examples/stocks/test/locale_test.dart

```diff
@@ -13,10 +13,13 @@ void main() {
     stocks.main();
     await tester.idle(); // see https://github.com/flutter/flutter/issues/1865
     await tester.pump();
+    // Note that the initial test app's locale is "_", so we're seeing
+    // the fallback translation here.
     expect(find.text('MARKET'), findsOneWidget);
     await tester.binding.setLocale('es', 'US');
     await tester.idle();
     await tester.pump();
+    await tester.pumpAndSettle(); // Wait for the localized resources to load.
```

Comment:
you don't need an "idle" and a "pump" and a "pumpAndSettle" in a row, the latter implies the first two.

But, you shouldn't need a pumpAndSettle. The point of this test having an "idle" and a "pump" is that it verifies that we never waste more than one frame to get the data ready here. It's a problem if it really is taking more than one.